app/assets/stylesheets/components/_page.css.sass

```diff
@@ -25,3 +21,8 @@
 
 .Page-item
   margin: $default-spacing
+
+@media only screen and (min-width : 640px)
+  .Page.Page--padding
+    .Page-content
+      padding: 16px
```

Comment:
`.Page-content` was added (in https://github.com/groupbuddies/council/pull/163) to replace `md-content`. 16px was the padding that `md-content` was using.